Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -200,11 +201,19 @@ public class JavaScriptLifter: ComponentBase, Lifter {
                 let dest = MemberExpression.new() <> input(0) <> "." <> op.propertyName
                 let expr = AssignmentExpression.new() <> dest <> " = " <> input(1)
                 w.emit(expr)
+                if options.contains(.collectTypes) {
```

Comment:
How about making a helper function (at the top of `lift`) named something like `maybeUpdateTypes` that contains this entire block so it's not repeated here and below?